File: client/src/app/_models/UserParams.ts
import { User } from "./user";

export class UserParams{
    gender:string
    orderBy:string = 'lastActive'

    minAge:number = 18
    maxAge:number = 80
    pageNumber:number = 1
    pageSize:number = 5
    constructor(user:User | null) {
        this.gender = user?.gender === 'female'? 'male' :'female'
        
    }
}

File: API/Helpers/UserParams.cs
﻿namespace API.Helpers;

public class UserParams:PaginationParams
{

    
    public string? Gender { get; set; }
    public string? CurrentUserName { get; set; }
    public int minAge { get; set; } = 18;
    public int maxAge { get; set; } = 100;
    
    public string? OrderBy { get; set; } 

}
